.gitignore

```diff
@@ -73,3 +73,7 @@ debug.log
 
 # Local Media Files
 .media/
+
+# Node modules & bower components
+junction/static/node_modules/
+junction/static/bower_components/
```

Comment:
Can you please add one empty line at the end of file every file as a editor guideline we follow. https://github.com/pythonindia/junction/blob/master/.editorconfig#L8. There are plenty of plugins/extensions available to make things easier. http://editorconfig.org :blush: